u.m

```diff
@@ -69,8 +69,8 @@
     % Establishes base unit system and preferences based on baseUnitSystem and
     % displayUnits.
     
-    baseUnitSystem =    evalin('base','baseUnitSystem')
-    dispUnits =         evalin('base','displayUnits')
+    baseUnitSystem =    evalin('caller','baseUnitSystem')
```

Comment:
I don't understand the purpose of this change. It seems like there might be some odd behavior is there's a "local" base unit system and a "global" base unit system that could be in conflict.

Reply:
You can notice the difference by running 'clc;addpath('./.tests/');runAll' on repository root.
In particular if you run the tests form some script, the base units will differ from your system defaults. Compare the following logs from running the tests on my system with 'baseUnitSystem =    evalin('base','baseUnitSystem')' and 'baseUnitSystem =    evalin('caller','baseUnitSystem')'
[run on windows with base.log](https://github.com/sky-s/physical-units-for-matlab/files/10528174/run.on.windows.with.base.log)
[run on windows with caller.log](https://github.com/sky-s/physical-units-for-matlab/files/10528175/run.on.windows.with.caller.log)

'''
================================================================================
Error occurred in testScript_noBaseUnits/Normal and it did not run to completion.
    ---------
    Error ID:
    ---------
    'DimVar:incompatibleUnits'
    --------------
    Error Details:
    --------------
    Error using  == 
    Incompatible units. Cannot perform operation on variables with different units.
    
    Error in testScript_noBaseUnits (line 10)
    assert(1==u.m)
================================================================================
================================================================================
Error occurred in testScript_noBaseUnits/Offset and it did not run to completion.
    ---------
    Error ID:
    ---------
    'DimVar:incompatibleUnits'
    --------------
    Error Details:
    --------------
    Error using  == 
    Incompatible units. Cannot perform operation on variables with different units.
    
    Error in testScript_noBaseUnits (line 15)
    assert(278.15==5*u.degC)
================================================================================
Failure Summary:

     Name                           Failed  Incomplete  Reason(s)
    ==============================================================
     testScript_noBaseUnits/Normal    X         X       Errored.
    --------------------------------------------------------------
     testScript_noBaseUnits/Offset    X         X       Errored.
'''
... while it runs trough normally if you scecify evaluation in caller warkspace